docs/typescript/reference/tools/cli.md

```diff
@@ -81,18 +78,6 @@ You must compile your code (`npm run build`) and start the debug proxy before ru
 - `-l, --loglevel <log-level>`: The severity of log entries emitted. Can be one of `debug`, `info`, `warn`, `error`, `emerg`, `crit`, `alert`.
 - `-d, --appDir <application-directory>`: The path to your application root directory.
 
----
-
-### `npx dbos-openapi generate`
```

Comment:
If we're adding this back: https://github.com/dbos-inc/dbos-transact-ts/pull/715
Do we still want to delete this reference?